_unittest/test_08_Primitives3D.py

```diff
@@ -701,12 +701,11 @@ def test_44_create_polyline_basic_segments(self):
         assert prim3D.create_polyline(
             position_list=test_points, segment_type=PolylineSegment("Spline", num_points=3), name="PL03_spline_3pt"
         )
-        try:
+        with pytest.raises(ValueError) as execinfo:
             prim3D.create_polyline(position_list=test_points[0:3], segment_type="Spline", name="PL03_spline_str_3pt")
-        except ValueError as e:
-            assert str(e) == "The position_list argument must contain at least 4 points for segment of type Spline."
-        else:
-            assert False
+            assert (
+                str(execinfo) == "The position_list argument must contain at least 4 points for segment of type Spline."
```

Comment:
```suggestion
                str(execinfo) == "The `position_list` argument must contain at least four points for segment of type `Spline`."
```
Usually single quotes surround code entities in message strings.